src/workerd/io/actor-cache-test.c++

```diff
@@ -4856,5 +4857,210 @@ KJ_TEST("ActorCache alarm delete when flush fails") {
   }
 }
 
+KJ_TEST("ActorCache can wait for flush") {
```

Comment:
It looks like this doesn't cover the case where one operations schedules a flush, then the flush starts, and then another operation schedules a second flush while the first flush is still in progress. It'd be worth confirming that:

1. an `onNoPendingFlush()` from before the second operation resolves when the first flush is resolved
2. an `onNoPendingFlush()` from after the second operation schedules a flush but before the first flush completes does not resolve until after the second flush completes.

Reply:
That makes sense to me as an additional thing to test, thanks! The set of combinations for this is starting to feel a bit nebulous, so I'll just use a put as the second flushable.